src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -27,21 +51,21 @@ existing published packages.
 
 ### Searching for packages
 
-Packages are published to [pub.dev][].
+Packages are published to the [pub][] site.
```

Comment:
We should talk about this terminology... it doesn't match what go/dart-terms currently says.